src/entities/User.ts

```diff
@@ -0,0 +1,46 @@
+import { Authorized, Field, ObjectType, ID } from 'type-graphql';
+import { Entity, PrimaryGeneratedColumn, CreateDateColumn, Column, UpdateDateColumn, OneToMany } from 'typeorm';
+
+import Item from './Item';
+
+@Entity()
+@ObjectType()
+class User {
+  @Authorized()
+  @Field(() => ID)
+  @PrimaryGeneratedColumn('uuid')
+  public uuid!: string;
```

Comment:
Would it be better to just name it as `id` rather than `uuid`?

Reply:
By my personal opinion,
I think `id` is a serialized and auto-ascending `number`, however, `uuid` is an unique `string`, not a serialized number.

How do you think this?

Reply:
I think `id` is much common API naming.  `uuid` is just for the technical terminology

Reply:
Ok, got it!